istioctl/pkg/multicluster/remote_secret.go

```diff
@@ -583,9 +583,9 @@ func (o *RemoteSecretOptions) addFlags(flagset *pflag.FlagSet) {
 		"Create a secret with this service account's credentials. Default value is \""+
 			constants.DefaultServiceAccountName+"\" if --type is \"remote\", \""+
 			constants.DefaultConfigServiceAccountName+"\" if --type is \"config\".")
-	flagset.BoolVar(&o.CreateServiceAccount, "create-service-account", true,
+	flagset.BoolVar(&o.CreateServiceAccount, "create-service-account", false,
```

Comment:
Didn't we agree that we don't want to change the defaults? I'd say keep the default, but return an error if this is set to true (we're effectively forcing the user to always explicitly set this to false).